content/developer/reference/backend/orm.rst

```diff
@@ -968,32 +971,56 @@ A domain is a list of criteria, each criterion being a triple (either a
         :attr:`~odoo.models.Model._parent_name`).
     ``any``
         matches if any record in the relationship traversal through
-        ``field_name`` (:class:`~odoo.fields.Many2one`,
+        ``field_expr`` (:class:`~odoo.fields.Many2one`,
         :class:`~odoo.fields.One2many`, or :class:`~odoo.fields.Many2many`)
         satisfies the provided domain ``value``.
+        The ``field_expr`` should be a field name.
     ``not any``
         matches if no record in the relationship traversal through
-        ``field_name`` (:class:`~odoo.fields.Many2one`,
+        ``field_expr`` (:class:`~odoo.fields.Many2one`,
         :class:`~odoo.fields.One2many`, or :class:`~odoo.fields.Many2many`)
         satisfies the provided domain ``value``.
 
 * ``value``
     variable type, must be comparable (through ``operator``) to the named
     field.
 
-Domain criteria can be combined using logical operators in *prefix* form:
+The :class:`~odoo.fields.Domain` can be used as a builder for domains.
```

Comment:
Either
```suggestion
:class:`~odoo.fields.Domain` can be used as a builder for domains.
```
but then it'd be better to auto-document it, or
```suggestion
The :class:`~odoo.fields.Domain` class can be used as a builder for domains.
```
or 
```suggestion
The :class:`~odoo.fields.Domain` constructor can be used as a builder for domains.
```
or
```suggestion
The :class:`~odoo.fields.Domain` API can be used as a builder for domains.
```
... you get it.